app/src/main/java/com/lightricks/feedexercise/network/DataTransferObjects.kt

```diff
@@ -3,3 +3,20 @@ package com.lightricks.feedexercise.network
 /**
  * todo: add Data Transfer Object data class(es) here
  */
+
+data class TemplatesMetadataItem(
+        val configuration: String,
+        val id: String,
+        val isNew: Boolean,
+        val isPremium: Boolean,
+        val templateCategories: List<String>,
+        val templateName: String,
+        val templateThumbnailURI: String
+)
+
+data class MetadataItemsList(
+        val templatesMetadata: List<TemplatesMetadataItem>
+)
```

Comment:
Please name the properties with the `JsonName` annotation so it will be clear from the code that we cannot rename those properties as we want without considering the json structure we get back from the web service. 

Also without doing this, it will not work in release variant of the app (production) since proguard/r8 is about to shrink these properties to a short name/obfuscated name. By annotating them moshi will know what the real name they have on the xml.

Last point generate an adapter by compile time for those classes so we will not do the parsing by using reflection that is slow.